src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,43 @@
 import React from 'react';
+import { Post } from '../../types/Post';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+type Props = {
+  post: Post,
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    body,
+    comment,
+    title,
+    user,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        <p>
+          {' Posted by  '}
+
+          {user && <UserInfo user={user} />}
+        </p>
+      </div>
+
+      <p className="PostInfo__body">
+        {body}
+      </p>
+
+      <>
+        <hr />
+        {!comment.length
+          ? (<b data-cy="NoCommentsMessage">No comments yet</b>)
+          : (<CommentList comment={comment} />)}
+      </>
```

Comment:
Seems that this empty fragment is redundant here. And this syntax for a ternary operator looks better to me 
```suggestion
        <hr />
        {!comment.length ? (
          <b data-cy="NoCommentsMessage">No comments yet</b>
        ) : (
          <CommentList comment={comment} />
        )}
```